### DISCUSSION TITLE
Downloading fire data in personal vector analysis

### DISCUSSION THREAD

**Discussion by Adriannaluger** [earthlab-education/Earth-Analytics-2023-01-Intro#494] [Q&A06 Vector Data]
Hi, I am starting my own vector analysis and want to look at the National Hydrology Dataset (HU4) and have successfully downloaded the data to my codespace. However, now I need to download fire data and am a bit confused on what data I should be downloading? From my understanding the fire needs to overlap my hydrology data in some areas but I am unsure of how to know if it does? Or is it just assumed that there will be some spots where overlap occurs? In that case would I just download the same fire data and continue my analysis? Thanks!

**Comment by eculler** [comment]
@Adriannaluger the fire data that we downloaded is national, so your download will not change if you are using HU4 watersheds vs. HU2.
